app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
```

Comment:
Couple moments:
test for membership would be "vaccine" in vstr;
more appropriate check if an object is None is is) like vsts["vaccine"] is None.
I'll suggest something. Will it work here?
```suggestion
        if not vstr.get('vaccine'):
```